src/data/actions/assets.js

```diff
@@ -41,7 +46,7 @@ export const getAssets = (request, courseDetails) =>
         }
       })
       .catch((error) => {
-        dispatch(assetDeleteFailure(error));
+        dispatch(requestAssetsFailure(error));
```

Comment:
I have changed this to be a requestAssetsFailure, since it's a failure to request assets and not a failure to delete an asset.